### ISSUE TITLE
Include error for Homebrew installed libraries

### ISSUE THREAD

**Issue by tierriminator** [microsoft/vscode-cpptools#8377]
Bug type: Language Service

**Describe the bug**
- OS and Version: MacOS 12.0.1
- VS Code Version: 1.61.2
- C/C++ Extension Version: 1.7.1

IntelliSense reports an error when including system headers installed via Homebrew, which are symlinked header files inside `/usr/local/include`.
Interestingly enough, code completion is able to find those headers.

**Steps to reproduce**
1. Install some library with Homebrew (e.g. libomp)
2. Try to include the respective header within a cpp file in VSCode (e.g. `#include <omp.h>`)
3. The line will be underlined red. Hovering over it will report the error (e.g. `cannot open source file "omp.h"`)

**Expected behavior**
No error should be reported

**Code sample and logs**
- Configurations in `c_cpp_properties.json`
```
{
    "configurations": [
        {
            "name": "Mac",
            "includePath": [
                "/usr/local/include"
            ],
            "defines": [],
            "macFrameworkPath": [],
            "compilerPath": "/usr/bin/clang++",
            "cStandard": "c11",
            "cppStandard": "c++11",
            "intelliSenseMode": "${default}",
            "configurationProvider": "ms-vscode.cmake-tools",
            "compilerArgs": [
                "-Xpreprocessor",
                "-fopenmp"
            ],
            "mergeConfigurations": true
        }
    ],
    "version": 4
}
```
- Logs from running `C/C++: Log Diagnostics` from the VS Code command palette
```
-------- Diagnostics - 11/7/2021, 11:50:57 PM
Version: 1.7.1
Current Configuration:
{
    "name": "Mac",
    "includePath": [
        "/usr/local/include"
    ],
    "defines": [],
    "macFrameworkPath": [],
    "compilerPath": "/usr/bin/clang++",
    "cStandard": "c11",
    "cppStandard": "c++11",
    "intelliSenseMode": "${default}",
    "configurationProvider": "ms-vscode.cmake-tools",
    "compilerArgs": [
        "-Xpreprocessor",
        "-fopenmp"
    ],
    "mergeConfigurations": true,
    "compilerPathIsExplicit": true,
    "cStandardIsExplicit": true,
    "cppStandardIsExplicit": true,
    "intelliSenseModeIsExplicit": true
}
Custom browse configuration: 
{
    "browsePath": [
        "/users/administrator/documents/studium/master/semester 5/masterarbeit/sparse-dsft-cpp/cpp/cpp",
        "/users/administrator/documents/studium/master/semester 5/masterarbeit/sparse-dsft-cpp/cpp/src"
    ],
    "standard": "c++14",
    "compilerPath": "/usr/bin/clang++",
    "compilerArgs": [
        "-pthread",
        "-std=c++14",
        "-funroll-loops",
        "-O2",
        "-march=native",
        "-mavx2",
        "-fPIC",
        "-g",
        "-isysroot",
        "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.0.sdk"
    ]
}
Translation Unit Mappings:
[ /Users/Administrator/Documents/Studium/Master/Semester 5/Masterarbeit/sparse-dsft-cpp/ssftapprox/multiset_enumeration.cpp ]:
    /Users/Administrator/Documents/Studium/Master/Semester 5/Masterarbeit/sparse-dsft-cpp/ssftapprox/multiset_enumeration.cpp
Translation Unit Configurations:
[ /Users/Administrator/Documents/Studium/Master/Semester 5/Masterarbeit/sparse-dsft-cpp/ssftapprox/multiset_enumeration.cpp ]:
    Process ID: 13996
    Memory Usage: 155 MB
    Compiler Path: /usr/bin/clang++
    Includes:
        /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1
        /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/13.0.0/include
        /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include
        /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
    Frameworks:
        /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks
    Standard Version: c++14
    IntelliSense Mode: macos-clang-x64
    Other Flags:
        --clang
        --clang_version=110000
Total Memory Usage: 155 MB

------- Workspace parsing diagnostics -------
Number of files discovered (not excluded): 2247
```
- Logs from [the language server logging](https://code.visualstudio.com/docs/cpp/enable-logging-cpp#_enable-logging-for-the-language-server)
```
sending compilation args for /Users/Administrator/Documents/Studium/Master/Semester 5/Masterarbeit/sparse-dsft-cpp/ssftapprox/binding.cpp
  include: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1
  include: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/13.0.0/include
  include: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include
  include: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
  framework: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks
  define: __llvm__=1
  define: __clang__=1
  define: __clang_major__=13
  define: __clang_minor__=0
  define: __clang_patchlevel__=0
  define: __clang_version__="13.0.0 (clang-1300.0.29.3)"
  define: __GNUC__=4
  define: __GNUC_MINOR__=2
  define: __GNUC_PATCHLEVEL__=1
  define: __GXX_ABI_VERSION=1002
  define: __GNUG__=4
  define: __GXX_WEAK__=1
  define: __ATOMIC_RELAXED=0
  define: __ATOMIC_CONSUME=1
  define: __ATOMIC_ACQUIRE=2
  define: __ATOMIC_RELEASE=3
  define: __ATOMIC_ACQ_REL=4
  define: __ATOMIC_SEQ_CST=5
  define: __OPENCL_MEMORY_SCOPE_WORK_ITEM=0
  define: __OPENCL_MEMORY_SCOPE_WORK_GROUP=1
  define: __OPENCL_MEMORY_SCOPE_DEVICE=2
  define: __OPENCL_MEMORY_SCOPE_ALL_SVM_DEVICES=3
  define: __OPENCL_MEMORY_SCOPE_SUB_GROUP=4
  define: __PRAGMA_REDEFINE_EXTNAME=1
  define: __VERSION__="Apple LLVM 13.0.0 (clang-1300.0.29.3)"
  define: __STRICT_ANSI__=1
  define: __GXX_EXPERIMENTAL_CXX0X__=1
  define: __OBJC_BOOL_IS_BOOL=0
  define: __CONSTANT_CFSTRINGS__=1
  define: __EXCEPTIONS=1
  define: __GXX_RTTI=1
  define: __DEPRECATED=1
  define: __private_extern__=extern
  define: __OPTIMIZE__=1
  define: __ORDER_LITTLE_ENDIAN__=1234
  define: __ORDER_BIG_ENDIAN__=4321
  define: __ORDER_PDP_ENDIAN__=3412
  define: __BYTE_ORDER__=__ORDER_LITTLE_ENDIAN__
  define: __LITTLE_ENDIAN__=1
  define: _LP64=1
  define: __LP64__=1
  define: __CHAR_BIT__=8
  define: __SCHAR_MAX__=127
  define: __SHRT_MAX__=32767
  define: __INT_MAX__=2147483647
  define: __LONG_MAX__=9223372036854775807L
  define: __LONG_LONG_MAX__=9223372036854775807LL
  define: __WCHAR_MAX__=2147483647
  define: __WINT_MAX__=2147483647
  define: __INTMAX_MAX__=9223372036854775807L
  define: __SIZE_MAX__=18446744073709551615UL
  define: __UINTMAX_MAX__=18446744073709551615UL
  define: __PTRDIFF_MAX__=9223372036854775807L
  define: __INTPTR_MAX__=9223372036854775807L
  define: __UINTPTR_MAX__=18446744073709551615UL
  define: __SIZEOF_DOUBLE__=8
  define: __SIZEOF_FLOAT__=4
  define: __SIZEOF_INT__=4
  define: __SIZEOF_LONG__=8
  define: __SIZEOF_LONG_DOUBLE__=16
  define: __SIZEOF_LONG_LONG__=8
  define: __SIZEOF_POINTER__=8
  define: __SIZEOF_SHORT__=2
  define: __SIZEOF_PTRDIFF_T__=8
  define: __SIZEOF_SIZE_T__=8
  define: __SIZEOF_WCHAR_T__=4
  define: __SIZEOF_WINT_T__=4
  define: __SIZEOF_INT128__=16
  define: __INTMAX_TYPE__=long int
  define: __INTMAX_FMTd__="ld"
  define: __INTMAX_FMTi__="li"
  define: __INTMAX_C_SUFFIX__=L
  define: __UINTMAX_TYPE__=long unsigned int
  define: __UINTMAX_FMTo__="lo"
  define: __UINTMAX_FMTu__="lu"
  define: __UINTMAX_FMTx__="lx"
  define: __UINTMAX_FMTX__="lX"
  define: __UINTMAX_C_SUFFIX__=UL
  define: __INTMAX_WIDTH__=64
  define: __PTRDIFF_TYPE__=long int
  define: __PTRDIFF_FMTd__="ld"
  define: __PTRDIFF_FMTi__="li"
  define: __PTRDIFF_WIDTH__=64
  define: __INTPTR_TYPE__=long int
  define: __INTPTR_FMTd__="ld"
  define: __INTPTR_FMTi__="li"
  define: __INTPTR_WIDTH__=64
  define: __SIZE_TYPE__=long unsigned int
  define: __SIZE_FMTo__="lo"
  define: __SIZE_FMTu__="lu"
  define: __SIZE_FMTx__="lx"
  define: __SIZE_FMTX__="lX"
  define: __SIZE_WIDTH__=64
  define: __WCHAR_TYPE__=int
  define: __WCHAR_WIDTH__=32
  define: __WINT_TYPE__=int
  define: __WINT_WIDTH__=32
  define: __SIG_ATOMIC_WIDTH__=32
  define: __SIG_ATOMIC_MAX__=2147483647
  define: __CHAR16_TYPE__=unsigned short
  define: __CHAR32_TYPE__=unsigned int
  define: __UINTMAX_WIDTH__=64
  define: __UINTPTR_TYPE__=long unsigned int
  define: __UINTPTR_FMTo__="lo"
  define: __UINTPTR_FMTu__="lu"
  define: __UINTPTR_FMTx__="lx"
  define: __UINTPTR_FMTX__="lX"
  define: __UINTPTR_WIDTH__=64
  define: __FLT16_DENORM_MIN__=5.9604644775390625e-8F16
  define: __FLT16_HAS_DENORM__=1
  define: __FLT16_DIG__=3
  define: __FLT16_DECIMAL_DIG__=5
  define: __FLT16_EPSILON__=9.765625e-4F16
  define: __FLT16_HAS_INFINITY__=1
  define: __FLT16_HAS_QUIET_NAN__=1
  define: __FLT16_MANT_DIG__=11
  define: __FLT16_MAX_10_EXP__=4
  define: __FLT16_MAX_EXP__=16
  define: __FLT16_MAX__=6.5504e+4F16
  define: __FLT16_MIN_10_EXP__=(-4)
  define: __FLT16_MIN_EXP__=(-13)
  define: __FLT16_MIN__=6.103515625e-5F16
  define: __FLT_DENORM_MIN__=1.40129846e-45F
  define: __FLT_HAS_DENORM__=1
  define: __FLT_DIG__=6
  define: __FLT_DECIMAL_DIG__=9
  define: __FLT_EPSILON__=1.19209290e-7F
  define: __FLT_HAS_INFINITY__=1
  define: __FLT_HAS_QUIET_NAN__=1
  define: __FLT_MANT_DIG__=24
  define: __FLT_MAX_10_EXP__=38
  define: __FLT_MAX_EXP__=128
  define: __FLT_MAX__=3.40282347e+38F
  define: __FLT_MIN_10_EXP__=(-37)
  define: __FLT_MIN_EXP__=(-125)
  define: __FLT_MIN__=1.17549435e-38F
  define: __DBL_DENORM_MIN__=4.9406564584124654e-324
  define: __DBL_HAS_DENORM__=1
  define: __DBL_DIG__=15
  define: __DBL_DECIMAL_DIG__=17
  define: __DBL_EPSILON__=2.2204460492503131e-16
  define: __DBL_HAS_INFINITY__=1
  define: __DBL_HAS_QUIET_NAN__=1
  define: __DBL_MANT_DIG__=53
  define: __DBL_MAX_10_EXP__=308
  define: __DBL_MAX_EXP__=1024
  define: __DBL_MAX__=1.7976931348623157e+308
  define: __DBL_MIN_10_EXP__=(-307)
  define: __DBL_MIN_EXP__=(-1021)
  define: __DBL_MIN__=2.2250738585072014e-308
  define: __LDBL_DENORM_MIN__=3.64519953188247460253e-4951L
  define: __LDBL_HAS_DENORM__=1
  define: __LDBL_DIG__=18
  define: __LDBL_DECIMAL_DIG__=21
  define: __LDBL_EPSILON__=1.08420217248550443401e-19L
  define: __LDBL_HAS_INFINITY__=1
  define: __LDBL_HAS_QUIET_NAN__=1
  define: __LDBL_MANT_DIG__=64
  define: __LDBL_MAX_10_EXP__=4932
  define: __LDBL_MAX_EXP__=16384
  define: __LDBL_MAX__=1.18973149535723176502e+4932L
  define: __LDBL_MIN_10_EXP__=(-4931)
  define: __LDBL_MIN_EXP__=(-16381)
  define: __LDBL_MIN__=3.36210314311209350626e-4932L
  define: __POINTER_WIDTH__=64
  define: __BIGGEST_ALIGNMENT__=16
  define: __INT8_TYPE__=signed char
  define: __INT8_FMTd__="hhd"
  define: __INT8_FMTi__="hhi"
  define: __INT8_C_SUFFIX__=
  define: __INT16_TYPE__=short
  define: __INT16_FMTd__="hd"
  define: __INT16_FMTi__="hi"
  define: __INT16_C_SUFFIX__=
  define: __INT32_TYPE__=int
  define: __INT32_FMTd__="d"
  define: __INT32_FMTi__="i"
  define: __INT32_C_SUFFIX__=
  define: __INT64_TYPE__=long long int
  define: __INT64_FMTd__="lld"
  define: __INT64_FMTi__="lli"
  define: __INT64_C_SUFFIX__=LL
  define: __UINT8_TYPE__=unsigned char
  define: __UINT8_FMTo__="hho"
  define: __UINT8_FMTu__="hhu"
  define: __UINT8_FMTx__="hhx"
  define: __UINT8_FMTX__="hhX"
  define: __UINT8_C_SUFFIX__=
  define: __UINT8_MAX__=255
  define: __INT8_MAX__=127
  define: __UINT16_TYPE__=unsigned short
  define: __UINT16_FMTo__="ho"
  define: __UINT16_FMTu__="hu"
  define: __UINT16_FMTx__="hx"
  define: __UINT16_FMTX__="hX"
  define: __UINT16_C_SUFFIX__=
  define: __UINT16_MAX__=65535
  define: __INT16_MAX__=32767
  define: __UINT32_TYPE__=unsigned int
  define: __UINT32_FMTo__="o"
  define: __UINT32_FMTu__="u"
  define: __UINT32_FMTx__="x"
  define: __UINT32_FMTX__="X"
  define: __UINT32_C_SUFFIX__=U
  define: __UINT32_MAX__=4294967295U
  define: __INT32_MAX__=2147483647
  define: __UINT64_TYPE__=long long unsigned int
  define: __UINT64_FMTo__="llo"
  define: __UINT64_FMTu__="llu"
  define: __UINT64_FMTx__="llx"
  define: __UINT64_FMTX__="llX"
  define: __UINT64_C_SUFFIX__=ULL
  define: __UINT64_MAX__=18446744073709551615ULL
  define: __INT64_MAX__=9223372036854775807LL
  define: __INT_LEAST8_TYPE__=signed char
  define: __INT_LEAST8_MAX__=127
  define: __INT_LEAST8_FMTd__="hhd"
  define: __INT_LEAST8_FMTi__="hhi"
  define: __UINT_LEAST8_TYPE__=unsigned char
  define: __UINT_LEAST8_MAX__=255
  define: __UINT_LEAST8_FMTo__="hho"
  define: __UINT_LEAST8_FMTu__="hhu"
  define: __UINT_LEAST8_FMTx__="hhx"
  define: __UINT_LEAST8_FMTX__="hhX"
  define: __INT_LEAST16_TYPE__=short
  define: __INT_LEAST16_MAX__=32767
  define: __INT_LEAST16_FMTd__="hd"
  define: __INT_LEAST16_FMTi__="hi"
  define: __UINT_LEAST16_TYPE__=unsigned short
  define: __UINT_LEAST16_MAX__=65535
  define: __UINT_LEAST16_FMTo__="ho"
  define: __UINT_LEAST16_FMTu__="hu"
  define: __UINT_LEAST16_FMTx__="hx"
  define: __UINT_LEAST16_FMTX__="hX"
  define: __INT_LEAST32_TYPE__=int
  define: __INT_LEAST32_MAX__=2147483647
  define: __INT_LEAST32_FMTd__="d"
  define: __INT_LEAST32_FMTi__="i"
  define: __UINT_LEAST32_TYPE__=unsigned int
  define: __UINT_LEAST32_MAX__=4294967295U
  define: __UINT_LEAST32_FMTo__="o"
  define: __UINT_LEAST32_FMTu__="u"
  define: __UINT_LEAST32_FMTx__="x"
  define: __UINT_LEAST32_FMTX__="X"
  define: __INT_LEAST64_TYPE__=long long int
  define: __INT_LEAST64_MAX__=9223372036854775807LL
  define: __INT_LEAST64_FMTd__="lld"
  define: __INT_LEAST64_FMTi__="lli"
  define: __UINT_LEAST64_TYPE__=long long unsigned int
  define: __UINT_LEAST64_MAX__=18446744073709551615ULL
  define: __UINT_LEAST64_FMTo__="llo"
  define: __UINT_LEAST64_FMTu__="llu"
  define: __UINT_LEAST64_FMTx__="llx"
  define: __UINT_LEAST64_FMTX__="llX"
  define: __INT_FAST8_TYPE__=signed char
  define: __INT_FAST8_MAX__=127
  define: __INT_FAST8_FMTd__="hhd"
  define: __INT_FAST8_FMTi__="hhi"
  define: __UINT_FAST8_TYPE__=unsigned char
  define: __UINT_FAST8_MAX__=255
  define: __UINT_FAST8_FMTo__="hho"
  define: __UINT_FAST8_FMTu__="hhu"
  define: __UINT_FAST8_FMTx__="hhx"
  define: __UINT_FAST8_FMTX__="hhX"
  define: __INT_FAST16_TYPE__=short
  define: __INT_FAST16_MAX__=32767
  define: __INT_FAST16_FMTd__="hd"
  define: __INT_FAST16_FMTi__="hi"
  define: __UINT_FAST16_TYPE__=unsigned short
  define: __UINT_FAST16_MAX__=65535
  define: __UINT_FAST16_FMTo__="ho"
  define: __UINT_FAST16_FMTu__="hu"
  define: __UINT_FAST16_FMTx__="hx"
  define: __UINT_FAST16_FMTX__="hX"
  define: __INT_FAST32_TYPE__=int
  define: __INT_FAST32_MAX__=2147483647
  define: __INT_FAST32_FMTd__="d"
  define: __INT_FAST32_FMTi__="i"
  define: __UINT_FAST32_TYPE__=unsigned int
  define: __UINT_FAST32_MAX__=4294967295U
  define: __UINT_FAST32_FMTo__="o"
  define: __UINT_FAST32_FMTu__="u"
  define: __UINT_FAST32_FMTx__="x"
  define: __UINT_FAST32_FMTX__="X"
  define: __INT_FAST64_TYPE__=long long int
  define: __INT_FAST64_MAX__=9223372036854775807LL
  define: __INT_FAST64_FMTd__="lld"
  define: __INT_FAST64_FMTi__="lli"
  define: __UINT_FAST64_TYPE__=long long unsigned int
  define: __UINT_FAST64_MAX__=18446744073709551615ULL
  define: __UINT_FAST64_FMTo__="llo"
  define: __UINT_FAST64_FMTu__="llu"
  define: __UINT_FAST64_FMTx__="llx"
  define: __UINT_FAST64_FMTX__="llX"
  define: __USER_LABEL_PREFIX__=_
  define: __FINITE_MATH_ONLY__=0
  define: __GNUC_GNU_INLINE__=1
  define: __GCC_ATOMIC_TEST_AND_SET_TRUEVAL=1
  define: __CLANG_ATOMIC_BOOL_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR16_T_LOCK_FREE=2
  define: __CLANG_ATOMIC_CHAR32_T_LOCK_FREE=2
  define: __CLANG_ATOMIC_WCHAR_T_LOCK_FREE=2
  define: __CLANG_ATOMIC_SHORT_LOCK_FREE=2
  define: __CLANG_ATOMIC_INT_LOCK_FREE=2
  define: __CLANG_ATOMIC_LONG_LOCK_FREE=2
  define: __CLANG_ATOMIC_LLONG_LOCK_FREE=2
  define: __CLANG_ATOMIC_POINTER_LOCK_FREE=2
  define: __GCC_ATOMIC_BOOL_LOCK_FREE=2
  define: __GCC_ATOMIC_CHAR_LOCK_FREE=2
  define: __GCC_ATOMIC_CHAR16_T_LOCK_FREE=2
  define: __GCC_ATOMIC_CHAR32_T_LOCK_FREE=2
  define: __GCC_ATOMIC_WCHAR_T_LOCK_FREE=2
  define: __GCC_ATOMIC_SHORT_LOCK_FREE=2
  define: __GCC_ATOMIC_INT_LOCK_FREE=2
  define: __GCC_ATOMIC_LONG_LOCK_FREE=2
  define: __GCC_ATOMIC_LLONG_LOCK_FREE=2
  define: __GCC_ATOMIC_POINTER_LOCK_FREE=2
  define: __PIC__=2
  define: __pic__=2
  define: __FLT_EVAL_METHOD__=0
  define: __FLT_RADIX__=2
  define: __DECIMAL_DIG__=__LDBL_DECIMAL_DIG__
  define: __SSP__=1
  define: __nonnull=_Nonnull
  define: __null_unspecified=_Null_unspecified
  define: __nullable=_Nullable
  define: __GCC_ASM_FLAG_OUTPUTS__=1
  define: __code_model_small__=1
  define: __amd64__=1
  define: __amd64=1
  define: __x86_64=1
  define: __x86_64__=1
  define: __SEG_GS=1
  define: __SEG_FS=1
  define: __seg_gs=__attribute__((address_space(256)))
  define: __seg_fs=__attribute__((address_space(257)))
  define: __corei7=1
  define: __corei7__=1
  define: __tune_corei7__=1
  define: __REGISTER_PREFIX__=
  define: __NO_MATH_INLINES=1
  define: __AES__=1
  define: __PCLMUL__=1
  define: __LAHF_SAHF__=1
  define: __LZCNT__=1
  define: __RDRND__=1
  define: __FSGSBASE__=1
  define: __BMI__=1
  define: __BMI2__=1
  define: __POPCNT__=1
  define: __PRFCHW__=1
  define: __RDSEED__=1
  define: __ADX__=1
  define: __MOVBE__=1
  define: __FMA__=1
  define: __F16C__=1
  define: __FXSR__=1
  define: __XSAVE__=1
  define: __XSAVEOPT__=1
  define: __XSAVEC__=1
  define: __XSAVES__=1
  define: __CLFLUSHOPT__=1
  define: __SGX__=1
  define: __INVPCID__=1
  define: __AVX2__=1
  define: __AVX__=1
  define: __SSE4_2__=1
  define: __SSE4_1__=1
  define: __SSSE3__=1
  define: __SSE3__=1
  define: __SSE2__=1
  define: __SSE2_MATH__=1
  define: __SSE__=1
  define: __SSE_MATH__=1
  define: __MMX__=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_1=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_2=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_4=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_8=1
  define: __GCC_HAVE_SYNC_COMPARE_AND_SWAP_16=1
  define: __APPLE_CC__=6000
  define: __APPLE__=1
  define: __STDC_NO_THREADS__=1
  define: __apple_build_version__=13000029
  define: __weak=__attribute__((objc_gc(weak)))
  define: __strong=
  define: __unsafe_unretained=
  define: __DYNAMIC__=1
  define: _REENTRANT=1
  define: __ENVIRONMENT_MAC_OS_X_VERSION_MIN_REQUIRED__=120000
  define: __ENVIRONMENT_OS_VERSION_MIN_REQUIRED__=120000
  define: __MACH__=1
  define: __STDC__=1
  define: __STDC_HOSTED__=1
  define: __cplusplus=201402L
  define: __STDCPP_DEFAULT_NEW_ALIGNMENT__=16UL
  define: __STDCPP_THREADS__=1
  define: __STDC_UTF_16__=1
  define: __STDC_UTF_32__=1
  other: --clang
  other: --clang_version=110000
  stdver: c++14
  intelliSenseMode: macos-clang-x64
```

**Comment by Colengms** [comment]
Hi @tierriminator .  When using a `configurationProvider`, the other fields of the base configuration are not used.  So, the `/usr/local/include` you have specified there is not being applied.  Instead, the configuration provider is used exclusively to configure files.  Based on your logs, it doesn't look like CMake Tools provided a configuration for `bindings.cpp` and the configuration fell back to the 'browse config' provided by CMake Tools (which also does not refer to `/usr/local/include`).

I suspect you could get the behavior you expect by updating your CMakeLists.txt to refer to `/usr/local/include` (i.e. using `target_include_directories`).

**Comment by tierriminator** [comment]
Thank you very much for your answer! Indeed, after I removed the `configurationProvider`, everything worked as expected. Everything else seems to be an issue with CMake Tools, hence I can close this issue.
Meanwhile I was able to work around the issue by overriding `C_Cpp.default.systemIncludePath` in `settings.json` with the paths dumped by `clang++ -E -x c++ - -v < /dev/null`. I think it's interesting that this works, considering your explanation.
